vcd/resource_vcd_vapp_vm.go

```diff
@@ -232,6 +232,12 @@ var vappVmSchema = map[string]*schema.Schema{
 					DiffSuppressFunc: suppressCase,
 					Description:      "Network card adapter type. (e.g. 'E1000', 'E1000E', 'SRIOVETHERNETCARD', 'VMXNET3', 'PCNet32')",
 				},
+				"connect": {
+					Type:        schema.TypeBool,
+					Optional:    true,
+					Default:     true,
+					Description: "If the virtual machine is undeployed, this value specifies whether the NIC should be connected upon deployment. If the virtual machine is deployed, this value reports the current status of this NIC's connection, and can be updated to change that connection status.",
```

Comment:
The comment looks to be misaligned with how it actually works. Especially the sentence "If the virtual machine is deployed, this value reports the current status of this NIC's connection". As far as I see it does not have `Computed` flag so it will not report it. Perhaps it could be simplified - just to say that in essence "It defines if NIC is connected or not". Deeper explanation could be in website docs if needed.

Reply:
changed to yours variation. I used API one.

Reply:
Cool. To me it looks straight forward without the need to digest a long sentence.